onnxruntime/python/tools/transformers/models/mistral/optimize.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import logging
+import os
```

Comment:
can we combine this file with llama?

Reply:
Yes - added all the workflow into the convert_to_onnx.py script with a flag 'optimize_optimum'. Currently, Mistral export is only tested with optimum and I've been using that exported model in all mistral work, but this optimization pipeline should work with other optimum models as well.

Updated README.md with this info